### ISSUE TITLE
How to set a keybinding in insert mode with jj to enter normal mode in meow emacs

### ISSUE THREAD

**Issue by sathishmanohar** [meow-edit/meow#359]
I am using meow for the past week and I like it very much. In the vim world I usually map `jj` to quickly exit out of insert mode to normal mode. how can I achieve that in meow?

I was able to find a solution for evil mode https://stackoverflow.com/questions/10569165/how-to-map-jj-to-esc-in-emacs-evil-mode

But I can't find anything for meow.

**Comment by DogLooksGood** [comment]
Try this
```
(setq key-chord-two-keys-delay 0.5)
(key-chord-define meow-insert-state-keymap "jj" 'meow-insert-exit)
(key-chord-mode 1)
```

`meow-insert-state-keymap` is the keymap used in insert state.

**Comment by sathishmanohar** [comment]
I get the message of

Symbol's function definition is void: key-chord-define

**Comment by sathishmanohar** [comment]
I didn't know keychord was a package. Emacs newbie here. Works perfectly. Thanks!

**Comment by sathishmanohar** [comment]
`(setq key-chord-two-keys-delay 0.5)`

key delay significantly slowed down movements because it delayed every key press by half a second.

I fixed it with setting the delay to 0.02 along with key-seq package. This is how my config looks now

`  ; Enable keychord
  (require 'key-chord)
  (key-chord-mode 1)

  ; jj to exit normal mode
  (setq key-chord-two-keys-delay 0.02)
  (key-seq-define meow-insert-state-keymap "jj" 'meow-insert-exit)`
  
  Reference: https://emacs.stackexchange.com/questions/13814/does-a-key-chord-that-looks-to-the-first-character-exists
